### DISCUSSION TITLE
At the beginning of a game of chess, how many blank playing spaces are on the board?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#64] [Galaxy Trivia] [answered]
**Hint:** board 8x8

**Comment by AsafMeizner** [comment, marked as answer]
There are 4 rows empty rows each 8 spaces meaning 32 blank spaces on the board

**Reply by ParanoidUser** [reply]
You did it! 😋 Congrats on your Galaxy Brain badge!  
If you're willing to be a part of the history, feel free to open PR and add your name to the [change log](https://github.com/ParanoidUser/yolo/blob/main/docs/CHANGELOG.md).

**Comment by JustGimli** [comment]
There are 4 rows empty rows each 8 spaces meaning 32 blank spaces on the board
